components/script/dom/htmllinkelement.rs

```diff
@@ -194,13 +194,6 @@ impl VirtualMethods for HTMLLinkElement {
                     }
                 }
             },
-            &local_name!("media") => {
-                if string_is_stylesheet(&rel) {
-                    if let Some(href) = self.upcast::<Element>().get_attribute(&ns!(), &local_name!("href")) {
-                        self.handle_stylesheet_url(&href.value());
```

Comment:
My concern about this patch was that, I mean... we definitely need to do _something_ when the `media` attribute of the link changes. At least evaluating media queries again and updating the associated stylesheet's media list.

It seems we don't do that at all now... It also means that the `mutation.is_removal()` bit is wrong in the condition above this.

Anyway, this can be a followup, given it's severely broken anyway looks like.

r=me